### DISCUSSION TITLE
[Feature] <Hide Uploaded Photos from Users>

### DISCUSSION THREAD

**Discussion by hareenpatel14** [immich-app/immich#5681] [Feature Request]
### The feature

Hey! want to start with thank you for the beautiful app!!

As i was using i had noticed in the sharing feature that if i create a link for users to only be able to upload that users are still able to view the uploaded images. 

I was wondering if it would be doable to add a option to " hide uploaded photos from guest " or something ? 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by Extarys** [comment]
Oh, I posted a discussion asking about that not long after you!
https://github.com/immich-app/immich/discussions/6230#discussioncomment-10647155

<3

**Comment by Absent-Reaction** [comment]
This would be a great feature to have added. Nextcloud has this

#24655
